### PR TITLE
Create and style responsive web design for desktop

### PR DESCRIPTION
**Pull request by Javivy**
In this milestone I: 
  ✅ Create the structure for the desktop version
  ✅ Style the desktop version
  ✅ Make the page responsive for any device

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,92 +170,104 @@ <h3>About Me</h3>
...
         <h2>
           I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
         </h2>
         <ul class="form-ul">
-          <li class="form-li">
-            <input autocomplete="off" maxlength="30" type="text" name="name" id="name" placeholder="Full Name" required>
+          <li class="form-li fst-and-lst-nm-input">
+            <input autocomplete="off" maxlength="30" type="text" name="first-name" id="first-name" placeholder="First Name" required>
+            <input autocomplete="off" maxlength="30" type="text" name="last-name" id="last-name" placeholder="Last Name" required>
+          </li>
```
Comment (leonardodiasb) [inline index.html:225]: - [ ] In the project requirements there was a note mentioning that on templates with different input fields from mobile to desktop, the 'full name' input field should be used for both versions. Please remove those fields and use only the 'full name' field for the desktop too. 👍

File: styles/styles.css [hunk 2]
```diff
@@ -409,8 +455,197 @@ footer {
...
+
+  .ab-me-cards-container {
+    display: grid;
+    grid-template-columns: repeat(3, 33.33%);
+    margin: 0 3rem;
+  }
+
+  .about-me-card {
+    width: 95%;
+  }
```
Comment (leonardodiasb) [inline styles/styles.css:553]: - [ ] Your language card does not have the same alignment as the rest due to the different amount of items in its list. Try changing the `justify-content` and using margins to reposition the contents inside this card so they can have the same alignment as the Figma template.
 -Your design:
image
-Figma template:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Javivy, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Very responsive desktop view. ✔️
 - Great use of background images. ✔️ 
 - Matching the Figma template almost perfectly. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Javivy) [reply index.html:225]: May I leave the form like that? It's working, please let me know :))

**Review by frank1738 — APPROVED** [review]
Hi @Javivy , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good Design 👍 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**